app/javascript/components/hoursCalculator.vue

```diff
@@ -29,6 +31,8 @@ export default {
   name: "hoursCalculator",
   data: function () {
     return {
+      nonWorkDays: 0,
+      helperCaption: "Holiday and weekend dates are not calculated.",
```

Comment:
When does this get displayed?  How about "Holiday and weekend dates are not used in hours calculation."

Reply:
It now only gives an explanation after the user selects a date range or updates the field.